civihr_default_theme/scss/pages/_my-leave.scss

```diff
@@ -9,4 +9,11 @@
   .button-container {
     margin-bottom: 40px;
   }
+
+  .btn-group {
+
+    > .btn {
+      border-color: transparent;
```

Comment:
After changing the default `btn-group` the "Record New Button" was broken, this to fix that. Updated the PR DOC